projects/novo-elements/src/elements/picker/extras/base-picker-results/BasePickerResults.ts

```diff
@@ -382,6 +382,14 @@ export class BasePickerResults {
   }
 
   preselected(match) {
+    if (this.config.preselected) {
+      let preselectedFunc: Function = this.config.preselected;
+      return (
+        this.selected.findIndex((item) => {
```

Comment:
If you don't need the index other than to tell whether a value has been found in the array, it'd be more idiomatic to use `Array.prototype.some` with that callback